tests/lax_numpy_operators_test.py

```diff
@@ -731,6 +731,30 @@ def testSpacingSubnormals(self, dtype):
     self._CheckAgainstNumpy(np.spacing, jnp.spacing, args_maker, check_dtypes=True, tol=0)
     self._CompileAndCheck(jnp.spacing, args_maker, tol=0)
 
+  @jtu.ignore_warning(category=RuntimeWarning,
+                      message="divide by zero.*")
```

Comment:
nit: `message` could go on the same line to make this easier to read

Reply:
Thanks! Done.